### PR TITLE
Command Tests.

### PR DESCRIPTION
**Pull request by wystrzal**

### CODE REVIEW

File: SK.Application.UnitTests/Discussions/Commands/EditDiscussionCommandTest.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+        public EditDiscussionCommandTest()
+        {
+            id = new Guid();
+            dbSetDiscussion = new Mock<DbSet<Discussion>>();
+            context = new Mock<IApplicationDbContext>();
+            stringLocalizer = new Mock<IStringLocalizer<DiscussionsResource>>();
+
+            discussion = new Discussion { Id = id };
+            discussionDto = new DiscussionEditDto { Id = id, Title = "Title", Description = "Description" };
```
Comment (matdon90) [inline SK.Application.UnitTests/Discussions/Commands/EditDiscussionCommandTest.cs:39]: Try not to use real texts. You could use Bogus to fake data.

File: SK.Application.UnitTests/Events/Commands/EditEventCommandTest.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+        public EditEventCommandTest()
+        {
+            id = new Guid();
+            dbSetEvent = new Mock<DbSet<Event>>();
+            context = new Mock<IApplicationDbContext>();
+            stringLocalizer = new Mock<IStringLocalizer<EventsResource>>();
+            mapper = new Mock<IMapper>();
+
+            @event = new Event { Id = id };
```
Comment (matdon90) [inline SK.Application.UnitTests/Events/Commands/EditEventCommandTest.cs:41]: Maybe better idea is to name it eventEntity or something similar isntead @event?

File: SK.Application.UnitTests/Posts/Commands/EditPostCommandTest.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        private readonly PostEditDto postDto;
+
+        public EditPostCommandTest()
+        {
+            dbSetPost = new Mock<DbSet<Post>>();
+            context = new Mock<IApplicationDbContext>();
+            stringLocalizer = new Mock<IStringLocalizer<PostsResource>>();
+         
+            post = new Post { Id = id };
+            postDto = new PostEditDto { Id = id, Body = "Body" };
```
Comment (matdon90) [inline SK.Application.UnitTests/Posts/Commands/EditPostCommandTest.cs:38]: As above - use Bogus.

File: SK.Application.UnitTests/User/Commands/AddRoleToUserCommandTest.cs [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SK.Application.UnitTests.User.Commands
+{
+    public class AddRoleToUserCommandTest
+    {
```
Comment (matdon90) [inline SK.Application.UnitTests/User/Commands/AddRoleToUserCommandTest.cs:22]: Suggest as above not to use defined texts.

File: SK.Application.UnitTests/User/Commands/DeleteUserCommandTest.cs [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+using System.Security.Claims;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SK.Application.UnitTests.User.Commands
+{
+    public class DeleteUserCommandTest
+    {
+        private const string userName = "User";
```
Comment (matdon90) [inline SK.Application.UnitTests/User/Commands/DeleteUserCommandTest.cs:24]: As above.

File: SK.Application.UnitTests/User/Commands/RemoveRoleFromUserCommandTest.cs [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SK.Application.UnitTests.User.Commands
+{
+    public class RemoveRoleFromUserCommandTest
+    {
+        private const string userName = "User";
```
Comment (matdon90) [inline SK.Application.UnitTests/User/Commands/RemoveRoleFromUserCommandTest.cs:23]: As above.

File: SK.Application.UnitTests/Discussions/Commands/EditDiscussionCommandTest.cs [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+        }
```
Comment (matdon90) [inline SK.Application.UnitTests/Discussions/Commands/EditDiscussionCommandTest.cs:40]: Try not to use real texts. You could use Bogus to fake data.

File: SK.Application.UnitTests/Events/Commands/EditEventCommandTest.cs [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+{
+    public class EditEventCommandTest
+    {
+        private readonly Guid id;
+        private readonly Mock<DbSet<Event>> dbSetEvent;
+        private readonly Mock<IApplicationDbContext> context;
+        private readonly Mock<IStringLocalizer<EventsResource>> stringLocalizer;
+        private readonly Mock<IMapper> mapper;
+
+        private readonly Event @event;
```
Comment (matdon90) [inline SK.Application.UnitTests/Events/Commands/EditEventCommandTest.cs:30]: Maybe better idea is to name it eventEntity or something similar isntead @event?

File: SK.Application.UnitTests/Posts/Commands/EditPostCommandTest.cs (hunk 3 above)
Comment (matdon90) [inline SK.Application.UnitTests/Posts/Commands/EditPostCommandTest.cs:38]: As above - use Bogus.

**Review by matdon90 — COMMENTED** [review]
Understandably and clearly written. I don't have any categorical comments and I haven't found any bugs. I suggest a few minor changes to the places I have marked. Generally good job.

**Review by matdon90 — APPROVED** [review]
Good job! All small issues awas fully resolved.

File: SK.Application.UnitTests/Articles/Commands/CreateArticleCommandTest.cs [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+            dbSetArticle = new Mock<DbSet<Article>>();
+            context = new Mock<IApplicationDbContext>();
+            stringLocalizer = new Mock<IStringLocalizer<ArticlesResource>>();
+            mapper = new Mock<IMapper>();
+        }
+
+        [Test]
+        public async Task ShouldCallHandle()
+        {
+            //Arrange
```
Comment (UgzSourceCode) [inline SK.Application.UnitTests/Articles/Commands/CreateArticleCommandTest.cs:39]: Please remove comments like that

**Review by UgzSourceCode — APPROVED** [review]

**Pull request merged**